### ISSUE TITLE
PhpStorm plugin: pressing Ctrl+Alt+R,T generates error

### ISSUE THREAD

**Issue by slaFFik** [beyondcode/tinkerwell-community#397]
1. In PhpStorm 2020.3.2 install the Tinkerwell plugin.
2. Enter the license key into plugin settings.
3. Press `Ctrl+Alt+R,T` (in Windows 10) to run the code.
4. See the error in the result output:

```
Parse error: syntax error, unexpected '?', expecting variable (T_VARIABLE) in phar://C:/OpenServer/userdata/temp/tinkerwell/tinker_prepared.phar/vendor/symfony/console/Output/Output.php on line 40
```

**Comment by mpociot** [comment]
Which PHP version is used as the interpreter in PHPStorm? Looks like it's an older version

**Comment by slaFFik** [comment]
@mpociot, you are correct.

I had 7.0 selected as my local interpreter inside PhpStorm (I have different PHP versions available locally).
- 7.0 generates that error reported in the issue description
- 7.1-7.4 works fine.

Could you please add a PHP version check in your code to prevent such errors and report about the incompatible PHP version?

**Comment by mpociot** [comment]
> Could you please add a PHP version check in your code to prevent such errors and report about the incompatible PHP version?

Yeah, that's a good idea. We will implement this in a future version of the plugin.
